client/src/components/BookCard/BookCard.js

```diff
@@ -88,7 +89,7 @@ const isPathToCover = function (props, pathToCover) {
 }
 
 BookCard.propTypes = {
-  id: PropTypes.number,
+  id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
why do you accept strings and numbers? the id from the db is always a string!? 🤔